crates/ironrdp-pdu/src/rdp/headers.rs

```diff
@@ -505,3 +513,35 @@ bitflags! {
         const FLUSHED = 0x80;
     }
 }
+
+/// 2.2.3.1 Server Deactivate All PDU
+///
+/// [2.2.3.1]: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/8a29971a-df3c-48da-add2-8ed9a05edc89
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ServerDeactivateAll;
+
+impl PduDecode<'_> for ServerDeactivateAll {
+    fn decode(src: &mut ReadCursor<'_>) -> PduResult<Self> {
+        let length_source_descriptor = src.read_u16();
+        let _ = src.read_slice(length_source_descriptor.into());
+        Ok(Self)
```

Comment:
issue: `ensure_fixed_part_size!(in: src);` / `ensure_size!` missing (the fuzzer is failing because of this)